src/components/dropdown.jsx

```diff
@@ -38,6 +29,14 @@ class Dropdown extends React.Component {
     this.setState({ expanded: true });
   }
 
+  updateMediaQuery(mediaQuery) {
+    if (mediaQuery.matches) {
```

Comment:
Very minor, but this can be collapsed into `this.setState({ mediaQuery: mediaQuery.matches })`.